docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,312 @@
+library: {library_name}                                 # Optional. Example: mylibrary
+library_link: {library_link}                            # Optional. URI to user provided library.
+tags:
+- {tag_0}                                               # Optional. Example: audio
+- {tag_1}                                               # Optional. Example: automatic-speech-recognition
+owners:
+- organization: {organization_name}                     # Required. Example: BZK
+  oin: {oin}                                            # Optional. Example: 00000001003214345000
+  name: {owner_name}                                    # Optional. Example: John Doe
+  email: {owner_email}                                  # Optional. Example: johndoe@email.com
+  role: {owner_role}                                    # Optional. Example: Data Scientist.
+
+model-index:
+- name: {model_id}                                      # Required. Example: CatClassifier.
+  model: {model_uri}                                    # Required. URI to a repository containing the model file.
+  artifacs:
+  - {model_artifact}                                    # Optional. URI to relevant model artifacts, if applicable.
+  parameters:
+  - name: {parameter_name}                              # Optional. Example: "epochs".
+    dtype: {parameter_dtype}                            # Optional. Example "int".
+    value: {parameter_value}                            # Optional. Example 100.
+  results:
+  - task:
+      type: {task_type}                                 # Required. Example: image-classification.
+      name: {task_name}                                 # Optional. Example: Image Classification.
+    dataset:
+      - type: {dataset_type}                            # Required. Example: common_voice. Link to a repository containing the dataset
+        name: {dataset_name}                            # Required. Example: "Common Voice (French)". A pretty name for the dataset.
+        split: {split}                                  # Optional. Example "train".
+        features:
+         - {feature_name}                               # Optional. Example: "gender".
+        revision: {dataset_version}                     # Optional. Example: 5503434ddd753f426f4b38109466949a1217c2bb
+    metrics:
+    - type: {metric_type}                               # Required. Example: false-positive-rate. Use metric id from https://hf.co/metrics.
+      name: {metric_name}                               # Required. Example: "FPR wrt class 0 restricted to feature gender:0 and age:21".
+      dtype: {metric_dtype}                             # Required. Example: "float".
+      value: {metric_value}                             # Required. Example: 0.75.
+      class: {metric_output_class}                      # Optional. Example: "1". Only relevant for certain metrics such as for example false-positive-rate positive rate.
```

Comment:
Could give an example how we use this i.c.w. dtype and value? Or could we achieve logging the same information with `value` i.c.w. `dtype`?